packages/create-gatsby/src/require-utils.ts

```diff
@@ -0,0 +1,4 @@
+export const requireResolve = (
```

Comment:
ℹ️ Aims to mock `require.resolve` that is not yet chipped in jest: https://github.com/facebook/jest/issues/9543